tools/test/test.bats

```diff
@@ -8,7 +7,8 @@ setup() {
     load 'test_helper/bats-file/load'
 
     DIR="$( cd "$( dirname "${BATS_TEST_FILENAME}" )/.." >/dev/null 2>&1 && pwd )"
-    PATH="$DIR/:$PATH" 
+    export PATH="$DIR/:$PATH:/usr/local/bin/chainloop_bin"
+    export DO_NOT_TRACK=1
```

Comment:
thanks!

and btw, what organization are you running these tests against? cc/ @javirln

Reply:
demo org